app/views/devise/registrations/new.html.erb

```diff
@@ -0,0 +1,44 @@
+<h2>Sign up</h2>
```

Comment:
- [ ] As the previous reviewer mentioned this error is thrown in the terminal when I try signup a new user. Kindly fix it to allow user registration.

![Screenshot_20221103_191137](https://user-images.githubusercontent.com/25227022/199802036-f477757a-4314-4967-a4cd-f564a13f572d.png)